cache/ttl/cache.go

```diff
@@ -110,11 +134,25 @@ func (cache *Cache) Delete(key string) bool {
 	return true
 }
 
+// Range calls f on every key
+func (cache *Cache) Range(f func(key string, value interface{}) error) {
+	cache.mutex.Lock()
```

Comment:
Move `hasEvictOnError` ahead
```
if !cache.hasEvictOnError {
continue
}
```

Reply:
`Range()` will call f() on all keys in the map
1. if `hasAutoExpire` and key is expired, skip it
2. if `hasEvictOnError`, delete the key or not when f() fails